backend/src/jwt/encode.js

```diff
@@ -4,7 +4,7 @@ import CONFIG from './../config'
 // Generate an Access Token for the given User ID
 export default function encode(user) {
   const token = jwt.sign(user, CONFIG.JWT_SECRET, {
-    expiresIn: 24 * 60 * 60 * 1000, // one day
+    expiresIn: '1d',
```

Comment:
I think this is more human readable and doesn't need a comment to explain it, but how is this fixing the expiration date? both are valid, correct?

Reply:
The was a misconception about the meaning of this number. It's **seconds** not milliseconds. So the previous expiration was after 1000 days.